gene/etl/base.py

```diff
@@ -150,3 +154,108 @@ def get_seqrepo(self, seqrepo_dir: Path) -> SeqRepo:
         if not Path(seqrepo_dir).exists():
             raise NotADirectoryError(f"Could not find {seqrepo_dir}")
         return SeqRepo(seqrepo_dir)
+
+    def _set_cl_interval_range(self, loc: str, arm_ix: int, location: Dict) -> None:
+        """Set the Chromosome location interval range.
+
+        :param loc: A gene location
+        :param arm_ix: The index of the q or p arm for a given location
+        :param location: VRS chromosome location. This will be mutated.
+        """
+        range_ix = re.search("-", loc).start()  # type: ignore
+
+        start = loc[arm_ix:range_ix]
+        start_arm_ix = re.search("[pq]", start).start()  # type: ignore
+        start_arm = start[start_arm_ix]
+
+        end = loc[range_ix + 1:]
+        end_arm_match = re.search("[pq]", end)
+
+        if not end_arm_match:
+            # Does not specify the arm, so use the same as start"s
+            end = f"{start[0]}{end}"
+            end_arm_match = re.search("[pq]", end)
+
+        end_arm_ix = end_arm_match.start()  # type: ignore
+        end_arm = end[end_arm_ix]
+
+        if (start_arm == end_arm and start > end) or \
+                (start_arm != end_arm and start_arm == "p" and end_arm == "q"):
+            location["start"] = start
+            location["end"] = end
+        elif (start_arm == end_arm and start < end) or \
+                (start_arm != end_arm and start_arm == "q" and end_arm == "p"):
+            location["start"] = end
+            location["end"] = start
+
+    # Add back once VRS Chromosome Location is supported in 2.0-alpha
+    # def _get_chromosome_location(self, location: Dict, gene: Dict) -> Optional[Dict]:
+    #     """Transform a gene's location into a GeneChromosomeLocation.
+
+    #     :param location: A gene's location.
+    #     :param gene: A transformed gene record.
+    #     :return: If location is a valid VRS ChromosomeLocation, return a dictionary
+    #         containing the ChromosomeLocation. Else, return None.
+    #     """
+    #     if "chr" in location and "start" in location and "end" in location:
+    #         if location["start"] == "p" and location["end"] == "p":
+    #             location["start"] = "pter"
+    #             location["end"] = "cen"
+    #         elif location["start"] == "q" and location["end"] == "q":
+    #             location["start"] = "cen"
+    #             location["end"] = "qter"
+    #         try:
+    #             chr_location = GeneChromosomeLocation(
+    #                 chr=location["chr"],
+    #                 start=location["start"],
+    #                 end=location["end"]).model_dump()
+    #         except ValidationError as e:
+    #             logger.info(f"{e} for {gene['symbol']}")
+    #         else:
+    #             return chr_location
+    #     return None
+
+    def get_seq_id_aliases(self, sr: SeqRepo, seq_id: str) -> List[str]:
```

Comment:
@korikuzma since these are only used by the ETL classes themselves, can we make them private? a la 

```suggestion
    def _get_seq_id_aliases(self, sr: SeqRepo, seq_id: str) -> List[str]:
```